packages/editor/src/components/toolbar/Toolbar2.tsx

```diff
@@ -113,30 +113,61 @@ export default function Toolbar() {
   const anchorPosition = useHookstate({ left: 0, top: 0 })
   const anchorOpen = useHookstate(false)
 
-  const { projectName } = useMutableState(EditorState)
+  const { projectName, sceneName } = useMutableState(EditorState)
+
   const hasLocationWriteScope = useUserHasAccessHook('location:write')
   const permission = useProjectPermissions(projectName.value!)
   const hasPublishAccess = hasLocationWriteScope || permission?.type === 'owner' || permission?.type === 'editor'
 
   return (
     <>
       <div className="flex items-center justify-between bg-theme-primary">
-        <Button
-          variant="outline"
-          rounded="none"
-          startIcon={<PiSquaresFourThin />}
-          className="border-0 bg-transparent"
-          onClick={(event) => {
-            anchorOpen.set(true)
-            anchorPosition.set({ left: event.clientX - 5, top: event.clientY - 2 })
-            anchorEl.set(event.currentTarget)
-          }}
-        />
-        {/* TO BE ADDED
-        <div className="flex items-center gap-2.5 rounded-full bg-theme-surface-main p-0.5">
+        <div className="flex items-center">
+          <div className="ml-3 mr-6 cursor-pointer" onClick={onCloseProject}>
+            <img src="favicon-32x32.png" alt="iR Engine Logo" className={`h-7 w-7 opacity-50`} />
+          </div>
+          <Button
+            endIcon={<MdOutlineKeyboardArrowDown size="1em" className="-ml-3 text-[#A3A3A3]" />}
+            iconContainerClassName="ml-2 mr-1"
+            rounded="none"
+            startIcon={<RxHamburgerMenu size={24} className="text-[#9CA0AA]" />}
+            className="-mr-1 border-0 bg-transparent p-0"
+            onClick={(event) => {
+              anchorOpen.set(true)
+              anchorPosition.set({ left: event.clientX - 5, top: event.clientY - 2 })
+              anchorEl.set(event.currentTarget)
+            }}
+          />
+          {/* Hiding the Panel Menu until we know what it does */}
+          {/* <Button
+            endIcon={
+              <MdOutlineKeyboardArrowDown
+                size="1em"
+                className={`-ml-3 text-[#A3A3A3] `}
+                // onClick={toggleDropdown}
+              />
+            }
+            rounded="none"
+            startIcon={<PiSquaresFourThin size={24} className="text-[#9CA0AA]" />}
+            iconContainerClassName="ml-2 mr-1"
+            className="-ml-1 border-0 bg-transparent p-0"
+            onClick={(event) => {
+              // anchorOpen.set(true)
+              // anchorPosition.set({ left: event.clientX - 5, top: event.clientY - 2 })
+              // anchorEl.set(event.currentTarget)
+            }}
+          /> */}
+        </div>
+        {/* TO BE ADDED */}
+        {/* <div className="flex items-center gap-2.5 rounded-full bg-theme-surface-main p-0.5">
```

Comment:
We should remove these lines. This results in code bloat.
To add this line later, we can put it in a separate commit (and then overwrite without those lines) and cherry-pick it when the requirements comes in.

Reply:
sounds good!